src/dispatcher.js

```diff
@@ -82,7 +82,8 @@ export default class Dispatcher {
   }
 
   getEditableBlockByEvent (evt) {
-    return evt.target.closest(this.editableSelector)
+    // For some unknown reason `closest` does not always exist in Firefox 89 ¯\_(ツ)_/¯
+    return evt.target.closest ? evt.target.closest(this.editableSelector) : undefined
```

Comment:
I suppose this is caused by text nodes beimg the event target.
```js
return (evt.target.closest ? evt.target : evt.target.parentNode).closest(this.editableSelector)
```

Reply:
It seems that if you click outside of the iframe then the target is the `HTMLDocument` and there is no `closest()` or `parentNode`.

A few observations with "initial" interaction blocker:
- On click the function is not called. This is probably due to `preventDefault()` being called elsewhere.
- On blur within the iframe the function is not called.
- On blur outside the iframe the function is called with the iframe body as the target (and the component is not deselected).
- On blur to another component the function is not called (but is called for the focus on the other component).